app/channels/room_channel.rb

```diff
@@ -47,6 +47,16 @@ def add_video(data)
     MessageReservationJob.set(wait_until: video.video_start_time).perform_later("start_video", start_message, video.room)
   end
 
+  def exit_force(data)
+    target = User.find(data["user_id"])
+    RoomChannel.broadcast_to target,
+                             render_error_json("force exit")
+    BanReport.create! target: target,
+                       reporter: current_user,
```

Comment:
__[RuboCop]__

Layout/AlignHash: Align the elements of a hash literal if they span more than one line.


<!-- Comment by [SideCI](https://sideci.com) -->